tests/unit/ts_utils_spec.lua

```diff
@@ -1,41 +1,45 @@
 local tsutils = require "nvim-treesitter.ts_utils"
 
-describe("is_in_node_range", function()
-  local function test_is_in_node_range(line, col)
-    local node = {
-      range = function()
-        return unpack { 0, 3, 2, 5 }
-      end,
-    }
-    return tsutils.is_in_node_range(node, line, col)
+describe("tree_contains", function()
+  local tree = {
+    root = function()
+      local node = {
+        range = function()
+          return unpack { 0, 3, 2, 5 }
+        end,
+      }
+      return node
+    end,
+  }
```

Comment:
Interesting solution :wink:

Reply:
Actually, I wanted to create LanguageTree, or TSNode, but i had no idea how to do this :)

Luckily - table could be used as a mock :)